lib/survey_gizmo/rest_response.rb

```diff
@@ -18,6 +18,9 @@ def initialize(rest_response)
 
     # Handle really crappy [] notation in SG API, so far just in SurveyResponse
     (data.is_a?(Array) ? data : [data]).each do |datum|
+      # Fix for when SG API returns [null] as data
+      next if dataum.is_a?(NilClass)
+      
```

Comment:
But it still returns `[nil]` for the value returned in the response.

```ruby
>> Array([nil])
=> [nil]
```

We can change
```ruby
data.is_a?(Array) ? data : [data]
```
to `Array(data)` though